batcher/aligned-sdk/Cargo.toml

```diff
@@ -25,3 +25,4 @@ hex = "0.4.3"
 ciborium = "=0.2.2"
 serde_repr = "0.1.19"
 dialoguer = "0.11.0"
+clap = { version = "4.5.4", features = ["derive"] }
```

Comment:
The `derive` feature is what allows us to do derive `ValueEnum`?

Reply:
Added!